libs/cardano-protocol-tpraos/src/Cardano/Protocol/TPraos/BHeader.hs

```diff
@@ -454,11 +454,12 @@ lastAppliedHash (At lab) = BlockHash $ labHash lab
 bnonce :: BHBody c -> Nonce
 bnonce = mkNonceFromOutputVRF . VRF.certifiedOutput . bheaderEta
 
-makeHeaderView :: Crypto c => BHeader c -> BHeaderView
-makeHeaderView bh@(BHeader bhb _) =
+makeHeaderView :: Crypto c => BHeader c -> Maybe Nonce -> BHeaderView
+makeHeaderView bh@(BHeader bhb _) nonce =
```

Comment:
This needs an entry in the changelog, since it is a breaking change. No need to bump the version, since current version on master already includes other breaking changes

Reply:
https://github.com/IntersectMBO/cardano-ledger/commit/9188e902a7ddd7509567c10e88b9d9d2d865f964